src/js/components/Drop/__tests__/Drop-test.js

```diff
@@ -205,18 +207,61 @@ describe('Drop', () => {
     const actualRoot = getByTestId('drop').parentNode.parentNode.parentNode;
     expect(actualRoot).toBe(document.body);
   });
+});
+
+test('custom containerTarget', () => {
+  const target = document.createElement('div');
+  document.body.appendChild(target);
+  try {
+    const { getByTestId } = render(
+      <TestInput data-testid="drop" containerTarget={target} />,
+    );
+    const actualRoot = getByTestId('drop').parentNode.parentNode.parentNode;
+    expect(actualRoot).toBe(target);
+  } finally {
+    document.body.removeChild(target);
+  }
+});
 
-  test('custom containerTarget', () => {
-    const target = document.createElement('div');
-    document.body.appendChild(target);
-    try {
-      const { getByTestId } = render(
-        <TestInput data-testid="drop" containerTarget={target} />,
+describe('Drop', () => {
```

Comment:
We shouldn't be adding `describe('Drop')` here again. The test can live in the existing "describe" block.

Reply:
Resolved this in my commit.

Reply:
Thank you 🙏